app/src/main/res/values-vi/strings.xml

```diff
@@ -1,3 +1,251 @@
+    <string name="maternity_leave">Nghỉ thai sản (180.0 ngày)</string>
+    <string name="wife_labor_leave">Vợ sinh con (14.0 ngày)</string>
+    <string name="company_pay">Công ty trả</string>
+    <string name="insurance_coverage">Bảo hiểm trả</string>
+    <string name="select_date">Chọn ngày</string>
+    <string name="select_time">Chọn thời gian</string>
+    <string name="reason">Lý do</string>
+    <string name="create">Tạo mới</string>
+    <string name="created_at">Thời gian tạo</string>
+    <string name="confirm_create">Xác nhận</string>
+    <string name="from">Từ</string>
+    <string name="to">Đến</string>
+    <string name="pending">Đang chờ</string>
+    <string name="accepted">Chấp nhận</string>
+    <string name="rejected">Từ chối</string>
+    <string name="forwarded">Chuyển tiếp</string>
+    <string name="leave_type">Hình thức xin phép</string>
+    <string name="compensation_time">Đăng ký thời gian làm bù</string>
+    <string name="example_time_from">Ví dụ: 2017/05/20 17:30</string>
+    <string name="example_time_to">Ví dụ: 2017/05/20 18:30</string>
+    <string name="confirm_request_leave">Xác nhận tạo yêu cầu đi trễ, về sớm</string>
+    <string name="submit">Gửi đi</string>
+    <string name="next">Tiếp theo</string>
+    <string name="status">Trạng thái</string>
+    <string name="all">Tất cả</string>
+    <string name="month">Tháng</string>
+    <string name="date">Ngày:</string>
+    <string name="date_of_creation">Ngày tạo:</string>
+    <string name="day_session">Buổi</string>
+    <string name="time_request">Thời gian yêu cầu:</string>
+    <string name="request_ot">Yêu cầu làm ngoài giờ</string>
+    <string name="confirm_request_overtime">Xác nhận yêu cầu làm ngoài giờ</string>
+    <string name="request_off">Yêu cầu xin nghỉ</string>
+    <string name="confirm_request_off">Xác nhận xin nghỉ</string>
+    <string name="note">Lưu ý: \u0020</string>
+    <string name="am_pm">Buổi sáng: am - am;  Buổi chiều: pm - pm;  Cả ngày: am - pm</string>
+    <string name="off_have_salary_from">Nghỉ có lương từ</string>
+    <string name="off_have_salary_to">Nghỉ có lương đến</string>
+    <string name="off_no_salary_from">Nghỉ không lương từ</string>
+    <string name="off_no_salary_to">Nghỉ không lương đến</string>
+    <string name="off_have_salary_insurance_coverage">Nghỉ có lương - Bảo hiểm trả</string>
+    <string name="edit">Sửa</string>
+    <string name="delete">Xóa</string>
+    <string name="being_handled_by">Nhóm đang xử lý</string>
+    <string name="confirm_edit_request_off">Xác nhận sửa yêu cầu xin nghỉ</string>
+    <string name="confirm_edit_request_overtime">Xác nhận sửa yêu cầu làm ngoài giờ</string>
+    <string name="edit_request_off">Sửa yêu cầu xin nghỉ</string>
+    <string name="edit_profile">Chỉnh sửa thông tin cá nhân</string>
+    <string name="change_password">Đổi mật khẩu</string>
+    <string name="default_group_and_workspace">Nhóm và workspace mặc định</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.default_group_and_workspace` appears to be unused
 Unused resources make applications larger and slow down builds.